internal/tools/sqlc-pg-gen/main.go

```diff
@@ -23,7 +23,8 @@ SELECT p.proname as name,
   format_type(p.prorettype, NULL),
   array(select format_type(unnest(p.proargtypes), NULL)),
   p.proargnames,
-  p.proargnames[p.pronargs-p.pronargdefaults+1:p.pronargs]
+  p.proargnames[p.pronargs-p.pronargdefaults+1:p.pronargs],
+  CASE WHEN p.prokind = 'a' THEN TRUE ELSE FALSE END
```

Comment:
I mentioned this in the discussion we were having, but I thought it is worth mentioning here. 

We probably should do a `CASE WHEN p.prokind = 'a' AND p.name != 'COUNT ....'` since `COUNT` is an exception. 

Also, I don't think there needs to be a `CASE` since you should just be able to use `p.prokind = 'a' AND p.name != 'COUNT'` as a boolean expression